eternalcore-core/src/main/java/com/eternalcode/core/feature/teleport/TeleportTask.java

```diff
@@ -40,33 +40,20 @@ class TeleportTask implements Runnable {
 
     @Override
     public void run() {
-        List<Teleport> teleportsCopy = new ArrayList<>(this.teleportTaskService.getTeleports());
+        List<Teleport> teleports = new ArrayList<>(this.teleportTaskService.getTeleports());
```

Comment:
im wondering about removing this?

Reply:
https://github.com/EternalCodeTeam/EternalCore/pull/740#discussion_r1536636345